scripts/dockerfiles/Dockerfile.build

```diff
@@ -76,9 +78,8 @@ FROM builder as compile
 
 # Get Fluent Bit source code
 WORKDIR /tmp/fluent-bit-$FLB_VERSION/
-RUN git clone https://github.com/amazon-contributing/upstream-to-fluent-bit.git /tmp/fluent-bit-$FLB_VERSION/
+RUN git clone --single-branch --branch $FLB_VERSION ${FLB_REPOSITORY} /tmp/fluent-bit-$FLB_VERSION/
```

Comment:
IIUC, the upstream fluent-bit repo uses tags instead of branch, for each version. I think we either need to switch our fork to checking out tags to be consistent with upstream, or update this command to be conditional depending on the repo.

Reply:
Nice. I would suggest just testing it once with upstream repo, unless you've done it already

Reply:
Ha, I definitely did both tags/branches without issues on both upstream/fluent-bit repositories. For this CR I directly tested against the existing upstream/branch combo.